src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -118,6 +132,16 @@ BinaryCompactObject::BinaryCompactObject(
     time_dependence_ =
         std::make_unique<domain::creators::time_dependence::None<3>>();
   }
+  if (use_logarithmic_map_object_A_ and not excise_interior_A_) {
+    PARSE_ERROR(context,
+                "excise_interior_A must be true if "
```

Comment:
These variable names don't show up in the input file so they might be confusing to users. Could you expand the error message to make sure it's clear what the issue is if this is triggered from an input file?